x-pack/plugins/fleet/server/services/epm/elasticsearch/template/template.ts

```diff
@@ -415,8 +416,17 @@ export const updateCurrentWriteIndices = async (
   if (!templates.length) return;
 
   const allIndices = await queryDataStreamsFromTemplates(esClient, templates);
-  if (!allIndices.length) return;
-  return updateAllDataStreams(allIndices, esClient, logger);
+  const allUpdatablesIndices = allIndices.filter((indice) => {
+    if (indice.replicated) {
+      logger.warn(
```

Comment:
I'm not sure if this should be a warning if it is a valid configuration of a system? Maybe debug? (not 100% sure on this)

Reply:
I think a warning is nice here as we fail to update existing datastreams and the user could have to take action to update them.